libs/components/forms/src/lib/modules/checkbox/checkbox.component.html

```diff
@@ -1,101 +1,108 @@
-<span class="sky-checkbox-outer-wrapper">
-  <label
-    class="sky-checkbox-wrapper sky-switch"
-    [ngClass]="{
-      'sky-control-label-required': required,
-      'sky-switch-disabled': disabled,
-      'sky-margin-inline-xs': labelText && !icon
-    }"
-  >
-    <input
-      class="sky-checkbox-input sky-switch-input"
-      type="checkbox"
-      [id]="inputId"
-      [checked]="checked"
-      [disabled]="disabled"
-      [name]="name"
-      [required]="required"
-      [tabIndex]="tabindex"
-      [attr.aria-label]="labelText || label"
-      [attr.aria-labelledby]="labelText ? undefined : labelledBy"
-      [attr.aria-describedby]="hintText ? hintTextEl.id : undefined"
-      [attr.aria-required]="required ? true : null"
-      [attr.aria-invalid]="!!ngControl?.errors"
-      [attr.aria-errormessage]="
-        labelText && ngControl?.errors ? errorId : undefined
-      "
-      [ngClass]="{
-        'sky-switch-invalid':
-          (ngControl?.touched || ngControl?.dirty) && ngControl?.errors
-      }"
-      (blur)="onInputBlur()"
-      (change)="onInteractionEvent($event)"
-      #inputEl
-    />
-    <span
-      class="sky-switch-control sky-rounded-corners"
+<span
+  class="sky-checkbox-container"
+  [ngClass]="{
+    'sky-checkbox-stacked': stacked
+  }"
+>
+  <span class="sky-checkbox-outer-wrapper">
+    <label
+      class="sky-checkbox-wrapper sky-switch"
```

Comment:
unfortunately I don't think so--they all wrap different layers of the component that all need to be wrapped for styling, the first two are just unfortunately very generically named, but I thought we couldn't change class names because consumers might be using them in tests?